docs/notebooks/doc2vec-IMDB.ipynb

```diff
@@ -610,7 +610,10 @@
     "        duration = 'na'\n",
     "        train_model.alpha, train_model.min_alpha = alpha, alpha\n",
     "        with elapsed_timer() as elapsed:\n",
-    "            train_model.train(doc_list, total_examples=train_model.corpus_count, epochs=train_model.iter)\n",
+    "            if not isinstance(train_model, ConcatenatedDoc2Vec):\n",
+    "              train_model.train(doc_list, total_examples=train_model.corpus_count, epochs=train_model.iter)\n",
+    "            else:\n",
+    "              train_model.train(doc_list)\n",
```

Comment:
Yes, you are right.  If the train() method is modified, the total_examples and epochs should be provided. But the ConcatenatedDoc2Vec class has no attribute 'corpus_count'.